packages/web3torrent/src/clients/payment-channel-client.ts

```diff
@@ -91,6 +94,17 @@ export class PaymentChannelClient {
     await this.channelClient.provider.enable();
 
     log.info('payment channel client enabled');
+
+    const doesBudgetExist = async () => {
+      const budget = await this.getBudget();
+      return !!budget && !_.isEmpty(budget);
+    };
+
+    if (FUNDING_STRATEGY !== 'Direct' && !(await doesBudgetExist())) {
+      // TODO: This only checks if a budget exists, not if we have enough funds in it
+      log.info('Direct Funding - Creating Budget');
```

Comment:
Is this actually virtual funding?
```suggestion
      log.info('Virtual Funding - Creating Budget');
```